app/actions/action_pins.js

```diff
@@ -1,37 +1,40 @@
 import { DROP_NEW_PIN } from '../constants/constants.js';
-import {store } from 'redux';
+import { store } from 'redux';
 let nextPinId = 0;
 
-
-// const fire = new Firebase("https://interruptedlobster.firebaseio.com/user1");
-
 function dropNewPin(pinInfo) {
   return {
     type: DROP_NEW_PIN,
     id: nextPinId++,
-    long: pinInfo.long,
-    lat: pinInfo.lat,
+    longitude: pinInfo.longitude,
+    latitude: pinInfo.latitude,
     title: pinInfo.title
```

Comment:
@bignell suggests:
Refactor to 
function dropNewPin( coords, title ) {
  return {
    type: DROP_NEW_PIN,
    id: nextPinId++,
    ...coords,
    title,
  }
}